src/cbor.h

```diff
@@ -452,21 +452,6 @@ CBOR_INLINE_API CborError cbor_value_dup_byte_string(const CborValue *value, uin
     return _cbor_value_dup_string(value, (void **)buffer, buflen, next);
 }
 
-CBOR_PRIVATE_API CborError _cbor_value_get_string_chunk(const CborValue *value, const void **bufferptr,
-                                                        size_t *len, CborValue *next);
-CBOR_INLINE_API CborError cbor_value_get_text_string_chunk(const CborValue *value, const char **bufferptr,
-                                                           size_t *len, CborValue *next)
-{
-    assert(cbor_value_is_text_string(value));
-    return _cbor_value_get_string_chunk(value, (const void **)bufferptr, len, next);
-}
-CBOR_INLINE_API CborError cbor_value_get_byte_string_chunk(const CborValue *value, const uint8_t **bufferptr,
-                                                           size_t *len, CborValue *next)
-{
-    assert(cbor_value_is_byte_string(value));
-    return _cbor_value_get_string_chunk(value, (const void **)bufferptr, len, next);
-}
-
```

Comment:
Right now `_cbor_value_get_string_chunk`, `cbor_value_get_text_string_chunk`, and `cbore_value_get_byte_string_chunk` look like they are all in a public header have they been part of a release yet?  Is it possible that they are already used by other projects?

That is my major concern when removing the headers.

I can see that  `_cbor_value_get_string_chunk` was already moved to cborinternal_p.h which is probably where is should have been to start with.

Reply:
Those functions are new in 0.5. I'm removing them so I can redesign them for when we're reading from something like Zephyr's `net_buf`.